application/F3DStarter.cxx

```diff
@@ -798,45 +798,80 @@ int F3DStarter::Start(int argc, char** argv)
     this->Internals->ApplyPositionAndResolution();
 
     f3d::interactor& interactor = this->Internals->Engine->getInteractor();
+
+    interactor.addCommandCallback("load_previous_file_group",
+      [this](const std::string&) { return this->LoadRelativeFileGroup(-1); });
```

Comment:
should we add a warning if an argument is provided but not used?

Reply:
feels not necessary to me

Reply:
wth the new impl, I only check in callbacks that uses the args.